package.json

```diff
@@ -7,19 +7,19 @@
   "description": "Dat Desktop App",
   "author": "Dat Team <community@datproject.org>",
   "scripts": {
-    "start": "cross-env NODE_V=\"$(node -v)\" NODE_ENV=development electron .",
-    "pretest": "npm run lint",
-    "test": "npm run test:unit",
+    "start": "cross-env NODE_ENV=development npm-run-all -p watch start:dev",
+    "start:dev": "cross-env NODE_V=\"$(node -v)\" NODE_ENV=development electron .",
+    "watch": "webpack --watch --mode=development",
```

Comment:
Maybe "start:watch"

Reply:
I thought changing this will make some people angry ;-)
But I agree with `start:watch`.